R/drive_trash.R

```diff
@@ -0,0 +1,111 @@
+#' Trash file on Google Drive.
+#'
+#' @template file
+#' @template verbose
+#'
+#' @return Logical, indicating whether the trashing succeeded.
+#' @export
+#' @examples
+#' \dontrun{
```

Comment:
Propose you add an upload command so we can actually run examples without writing extra code. See the help for `drive_cp()` or `drive_upload()` for inspiration.